.github/workflows/Docker.yml

```diff
@@ -1,100 +0,0 @@
-name: Build and Test - Docker
```

Comment:
Removed as I am not sure what is the point of this. The tests are executed without this test runner.